tests/features/support/steps_clone.js

```diff
@@ -17,173 +18,173 @@ KEY:
 // const scope = require('./scope');
 
 // /* Of Note:
-// - We're using `*=` because sometimes da text has funny characters in it
-//     that are hard to anticipate, so we allow people to just put in some of the text
+// - We're using `*=` because sometimes da text has funny characters in it 
+//     that are hard to anticipate, so we allow people to just put in some of the text 
 // */
 
 
 // // =========================
 // // Contstants
 // // =========================
 // let ordinal_to_integer = {
-//   first: 0, second: 1, third: 2, fourth: 3, fifth: 4,
+//   first: 0, second: 1, third: 2, fourth: 3, fifth: 4, 
 //   sixth: 5, seventh: 6, eighth: 7, ninth: 8, tenth: 9,
 //   '1st': 0, '2nd': 1, '3rd': 2, '4th': 3, '5th': 4,
 //   '6th': 5, '7th': 6, '8th': 7, '9th': 8,'10th': 9,
 // };
 // let specified = '?"?(first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth|"[^"]+)?"?';
-// let click_with = { mobile: 'tap', pc: 'click', };
+// let click_with = { mobile: 'tap', pc: 'click', }; 
 // // non-capturing capturing group expression for single or double quotes?
 
 
 // =========================
 // Establishing
 // =========================
-//x @step(r'I spend at least ([0-9]+) seconds? on each page')
+//x @step(r'I spend at least ([0-9]+) seconds? on each page') 
 //x @step(r'I want to store screenshots in the folder "([^"]+)"')
 
 
 // =========================
 // Start
 // =========================
-//x @step(r'I start the possibly error-producing interview "([^"]+)"')
+//x @step(r'I start the possibly error-producing interview "([^"]+)"') 
 //x @step(r'I am using the server "([^"]+)"')
 //x @step(r'I log in with "([^"]+)" and "([^"]+)"')
-//x @step(r'I launch the interview "([^"]+)"')
+//x @step(r'I launch the interview "([^"]+)"') 
 
 // ? @step(r'I start the interview "([^"]+)"')
 // ? I start the interview file "([^"]+)" on mobile
-// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???)
+// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???) 
 // ? I open the interview file ?"?([^"]+)"?[ on ]?(.*)/
-// √ I start the interview on mobile|pc
+// √ I start the interview on mobile|pc 
```

Comment:
Another possibility to have the dev name the file might be `I start the interview called "" on mobile`. Does that give the impression it's looking for a file name, though? Or a form/title name or what?

Reply:
It's unclear what it's looking for from that sentence. People will either have to rely on documentation/prior examples to figure out it needs a file name, or we can be more explicit (but less normal-Englishy) by saying `I start the interview in the file "" on mobile.`